pkg/pgmodel/ingestor/inserter.go

```diff
@@ -107,11 +108,11 @@ func (p *pgxInserter) refreshSeriesEpoch(existingEpoch model.SeriesEpoch) model.
 		// and continue execution
 		log.Error("msg", "error refreshing the series cache", "err", err)
```

Comment:
(optional) I think we should always do bare logging from the parent, and from a child. So, this should return an error and `runSeriesEpochSync` would be responsible for logging it out.